### PR TITLE
Updates to the wof-data README

### PR DESCRIPTION
**Pull request by stepps00**
Various updates to the README.md file for this repo. Mostly corrections to supplemental repo links, new blog post links, and minor text edits.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -235,9 +254,9 @@ https://whosonfirst.mapzen.com/spelunker/
 
 ## Venues
 
-For the time being Who's On First maintains a separate repository with venues and points-of-interest. It is:
+For the time being Who's On First maintains separate repositories with venues and points-of-interest. The starting point for venue data is:
 
-https://github.com/whosonfirst/whosonfirst-venue
+https://github.com/whosonfirst/whosonfirst-data-venue
```
Comment (nvkelso) [inline README.md]: This should be a pattern, with 2-3 examples (note also the org name should be -data):

https://github.com/whosonfirst-data/whosonfirst-data-venue-*

https://github.com/whosonfirst-data/whosonfirst-data-venue-us-ca
https://github.com/whosonfirst-data/whosonfirst-data-venue-us-ny
https://github.com/whosonfirst-data/whosonfirst-data-venue-ca

File: README.md [hunk 2]
```diff
@@ -256,7 +256,13 @@ https://whosonfirst.mapzen.com/spelunker/
 
 For the time being Who's On First maintains separate repositories with venues and points-of-interest. The starting point for venue data is:
```
Comment (nvkelso) [inline README.md]: We should note `*` should be replaced with country code &/or country code - region code.

Comment (thisisaaronland) [reply README.md]: When this is merged assign a ticket to me and I will document the model and rationale behind the naming conventions.

**Comment by stepps00** [comment]
https://github.com/whosonfirst-data/whosonfirst-data/pull/986#pullrequestreview-69733594 taken care of in: ca66f39. Merging and closing.

File: README.md (hunk 2 above)
Comment (nvkelso) [reply README.md]: See https://github.com/whosonfirst-data/whosonfirst-data/issues/998.

Comment (nvkelso) [reply README.md]: Err, the original is: https://github.com/whosonfirst-data/whosonfirst-data/issues/989

**Pull request merged**